luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -0,0 +1,514 @@
+    for id, power in pairs(teamPowers) do
+        if id == teamID then
+            return power
+        end
+    end
+    return 0
+end
+
+--Returns the total power of all non scavenger/raptor teams as a number.
+local function totalPlayerTeamsPower()
+    local totalPower = 0
+
+    for teamID, power in pairs(teamPowers) do
+        if teamID ~= neutralTeam and teamID ~= scavengerTeam and teamID ~= raptorTeam then
+            totalPower = totalPower + power
+        end
+    end
+
+    return totalPower
+end
+
+--Returns the highest non scavenger/raptor team power as a table {teamID, power}.
+local function highestPlayerTeamPower()
+    local highestPower = 0
+    local highestTeamID = nil
+
+    for teamID, power in pairs(teamPowers) do
+        if teamID ~= neutralTeam and teamID ~= scavengerTeam and teamID ~= raptorTeam then
+            if power > highestPower then
+                highestPower = power
+                highestTeamID = teamID
+            end
+        end
+    end
+
+    return {teamID = highestTeamID, power = highestPower}
+end
+
+--Returns the average of all non scavenger/raptor teams as a number.
+local function averagePlayerTeamPower()
+    local totalPower = 0
+    local teamCount = 0
+
+    for id, power in pairs(teamPowers) do
+        if id ~= neutralTeam and id ~= scavengerTeam and id ~= raptorTeam then
+            totalPower = totalPower + power
+            teamCount = teamCount + 1
+        end
+    end
+
+    local averagePower = totalPower / teamCount
+    return averagePower
+end
+
+--Returns the lowest non scavenger/raptor team power as a table {teamID, power}.
+local function lowestPlayerTeamPower()
+    local lowestPower = math.huge
+    local lowestTeamID = nil
+
+    for teamID, power in pairs(teamPowers) do
+        if teamID ~= neutralTeam and teamID ~= scavengerTeam and teamID ~= raptorTeam then
+            if power < lowestPower then
+                lowestPower = power
+                lowestTeamID = teamID
+            end
+        end
+    end
+
+    return {teamID = lowestTeamID, power = lowestPower}
+end
+
+--Returns the highest non AI/scavenger/raptor team power as a table {teamID, power}.
+local function highestHumanTeamPower()
+    local highestPower = 0
+    local highestTeamID = nil
+
+    for teamID, power in pairs(teamPowers) do
+        if humanTeams[teamID] then
+            if power > highestPower then
+                highestPower = power
+                highestTeamID = teamID
+            end
+        end
+    end
+
+    return {teamID = highestTeamID, power = highestPower}
+end
+
+--Returns the average of all non AI/scavenger/raptor teams as a number.
+local function averageHumanTeamPower()
+    local totalPower = 0
+    local teamCount = 0
+
+    for id, power in pairs(teamPowers) do
+        if humanTeams[id] then
+            totalPower = totalPower + power
+            teamCount = teamCount + 1
+        end
+    end
+
+    local averagePower = totalPower / teamCount
```

Comment:
```suggestion
    local averagePower = teamCount > 0 and totalPower / teamCount or 0
```
See comment on line 155